docs/content/en/docs-dev/user-guide/managing-controlplane/metrics.md

```diff
@@ -86,3 +86,17 @@ And give it to the `helm install` command when [installing](../../../installatio
 ```
 
 See [here](https://prometheus.io/docs/alerting/latest/configuration/) for more details on AlertManager's configuration.
+
+## Piped agent metrics
+
+| Metric | Type | Description |
+| --- | --- | --- |
+| `deployment_status` | gauge | The current status of deployment. 1 for current status, 0 for others. |
```

Comment:
How about other metrics of piped. We serve other as well
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/cmd/piped/piped.go#L815-L821

Reply:
Thanks! I missed them.